freqtrade/persistence/models.py

```diff
@@ -165,6 +165,16 @@ def update_from_ccxt_object(self, order):
                 self.order_filled_date = datetime.now(timezone.utc)
         self.order_update_date = datetime.now(timezone.utc)
 
+    def to_json(self) -> Dict[str, Any]:
+        return {
+            'cost': self.cost if self.cost else 0,
+            'amount': self.amount,
+            'price': self.price,
+            'average': round(self.average, 8) if self.average else 0,
+            'order_filled_date': self.order_filled_date.strftime(DATETIME_PRINT_FORMAT)
```

Comment:
I'd always use a "_timestamp" attribute here as well (aligned to how we do it for the `trade.to_json()`).

These methods (`to_json`) are intended to be directly used by the API - and parsing a date from a string in clients is quite errorprone (and sometimes expensive).

I'd also add some more fields like `side` (or better `ft_order_side`) and `order_type` as well as `filled` (that's some i know by memory - not necessarily all) - as this will not just be limited to telegram, but will also be used by the API - where it makes sense to show some of these.